### ISSUE TITLE
Migration pending after a fresh installation

### ISSUE THREAD

**Issue by XaviP** [barseghyanartur/django-fobi#35]
Hi, thanks for the software!
We're trying to integrate django-fobi in our project and we've found that after install it (pip install django-fobi) and run succesfully *python manage.py migrate*, there's still a pending migration:
```
python manage.py makemigrations

Migrations for 'fobi':
  0004_auto_20160905_2254.py:
    - Alter field plugin_uid on formelement
    - Alter field plugin_uid on formelemententry
    - Alter field plugin_uid on formhandler
    - Alter field plugin_uid on formhandlerentry
```
The migration file is created inside virtualenv:
env/lib/python2.7/site-packages/fobi/migrations/0004_auto_20160905_2254.py
You can see the migration file [here](https://gist.github.com/XaviP/d24d93d17e508570aa41b4c20fd7b05e).
We can run *python manage.py migrate* again without problems:
```
Running migrations:
  Rendering model states... DONE
  Applying fobi.0004_auto_20160905_2254... OK
```
But we think it's better to integrate the migration file into the package to keep the db sync when updating.
Thanks.

**Comment by XaviP** [comment]
Done a pull request with this migration file:
https://github.com/barseghyanartur/django-fobi/pull/36

**Comment by XaviP** [comment]
One of our developers reports no problems in version 0.6.6
This is only in version 0.6.7

**Comment by barseghyanartur** [comment]
@XaviP:

Thanks for reporting this. I don't remember making changes to the database. It might be that just the order of choices has been changed and Django sees it as a model change. I'll check it in any case ASAP.

**Comment by barseghyanartur** [comment]
@XaviP:

Fixed in version 0.6.8. I must reject this pull request, since I have came with a better fix. In any case, thanks for the efforts! Pull requests are always welcome.

**Comment by XaviP** [comment]
You are so fast attending our issues that we are happy and trustful with django-fobi.
Thank you very much !!
